content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -89,168 +89,138 @@ EOF
 sudo sysctl --system
 ```
 
-Install containerd:
+Install `containerd`:
 
 {{< tabs name="tab-cri-containerd-installation" >}}
 {{% tab name="Ubuntu 16.04" %}}
 
-```shell
-# (Install containerd)
-## Set up the repository
-### Install packages to allow apt to use a repository over HTTPS
-sudo apt-get update && sudo apt-get install -y apt-transport-https ca-certificates curl software-properties-common
-```
+1. Setup the [Docker Engine repository for Ubuntu](https://docs.docker.com/engine/install/ubuntu/#set-up-the-repository)
```

Comment:
i think we should remove the tabs here and just instruct the users to pick a distribution from here and setup their registry.
https://docs.docker.com/engine/install/#server

after that we can include a common list of commands.

Reply:
It looks like the Ubuntu 18.04/20.04 specifically differs from Ubuntu 16.04 and Debian 9+ tabs, in that it does not use the docker repository and it installs `containerd` (note not `containerd.io`) from the Ubuntu repositories and not the Docker ones - I'm not sure if this is deliberate? We will still need a tab for `Windows (Powershell)`, as even the mostly common tasks like `sudo systemctl restart containerd` won't apply to this tab, so we will need it regardless. If there is to be a clean up, would preferably like it in another PR, since I would not like to compound multiple refactoring tasks in the one PR.